clients/drcachesim/analysis_tool.h

```diff
@@ -455,12 +500,21 @@ template <typename RecordType> class analysis_tool_tmpl_t {
      * \p interval_id is a positive ordinal of the trace interval that just ended.
      * Trace intervals have a length equal to the \p -interval_microseconds specified
      * to the framework. Trace intervals are measured using the value of the
-     * #TRACE_MARKER_TYPE_TIMESTAMP markers. The provided \p interval_id
-     * values will be monotonically increasing but may not be continuous,
-     * i.e. the tool may not see some \p interval_id if the trace shard did not
-     * have any activity in that interval.
+     * #TRACE_MARKER_TYPE_TIMESTAMP markers.  Alternatively, trace intervals may be
+     * defined in terms of the \p -interval_instr_count count of shard-local instrs
+     * specified to the framework.
```

Comment:
Much of this comment appears identical to the comment text above, so my same comments apply about describing both interval modes together.  Also, duplicate descriptions like this are problematic, since it's so easy to update one and forget the other, and then end up with them out of sync.  Maybe one of the two could be condensed to minimal descriptions plus a pointer to the other more detailed comment.

Reply:
Done.